test/parallel/xyk-pallet.asset.buy.test.ts

```diff
@@ -1,6 +1,7 @@
 /*
  *
  * @group xyk
+ * @group market
```

Comment:
only thi schange on this file?? :sweat_smile:

Reply:
No-no, sure, this is the change from the last PR. Fixed